### PR TITLE
Ready for publishing to Github Packages

### PR DESCRIPTION
**Pull request by zijchen**

### CODE REVIEW

File: builds/build-release.yml [hunk 1]
```diff
@@ -27,7 +21,7 @@ stages:
     displayName: 'Release: Build, Test, and Publish on Windows'
 
     pool:
-      vmImage: 'windows-latest'
+      vmImage: 'ubuntu-latest'
```
Comment (zijchen) [inline builds/build-release.yml:24]: Building on Linux is 40% faster

**Review by kisantia — APPROVED** [review]

**Review by Benjin — APPROVED** [review]

File: builds/build-release.yml [hunk 2]
```diff
@@ -1,33 +1,27 @@
...
 
 pr: none
 
-schedules:
-  - cron: "0 0 * * *"
-    displayName: Mon-Fri at Midnight
-    branches:
-      include:
-        - main
-    always: true
```
Comment (Benjin) [inline builds/build-release.yml:11]: Why is the schedule removed?

Comment (zijchen) [reply builds/build-release.yml:11]: I don't see the benefit to running this every night, I could be convinced though 😄

Comment (Benjin) [reply builds/build-release.yml:11]: Nah, that's all good.  Is it now just on-demand, or is there a new trigger?

Comment (zijchen) [reply builds/build-release.yml:11]: Yeah just run manually now, the build number will still be incremented every time

File: src/Microsoft.Build.Sql/README.md [hunk 3]
```diff
@@ -7,36 +7,28 @@
...
-   dotnet new nugetconfig
-   ```
-2. Edit `nuget.config` by adding a package source pointing to the folder that contains Microsoft.Build.Sql.1.0.0.nupkg
-   ```xml
-    <packageSources>
-      <clear />
-      <add key="local" value="{PATH_OF_SDK}" />
-    </packageSources>
-   ```
+### Add Github Packages as Nuget Source
```
Comment (dzsquared) [inline src/Microsoft.Build.Sql/README.md:11]: for easier use of links by folks who are not English-language dominant, it's best practice to see if a docs link auto-resolves if you remove the language-specific segment.
eg: https://docs.github.com/packages/working-with-a-github-packages-registry/working-with-the-nuget-registry#authenticating-to-github-packages

nit: GitHub with capital H

**Review by dzsquared — APPROVED** [review]

**Pull request merged**